### DISCUSSION TITLE
outer legend type element around Joint JS Element - is it doable

### DISCUSSION THREAD

**Discussion by adarshjalaja** [clientIO/joint#2003] [Q&A]
is it possible to draw an outer element like HTML legend around joints JS element to add an extra label in addition to the
label inside element. or add legend type label on the joint JS element (in addition to text inside element)

**Comment by jamesgeorgewilliams** [comment]
There are some examples at the following link that may interest you:

https://jsfiddle.net/kumilingus/97dwxqv3/98/

The `TwoLabelsRectangle` shape shows a 2nd label/legend outside the main rectangle.

The `Fields` shape shows an "outer" element which contains another label.


<img width="278" alt="Screenshot 2023-01-23 at 11 43 15" src="https://user-images.githubusercontent.com/42288565/214020357-17defdbd-7748-4d31-a04f-8689e333dd0e.png">

**Reply by adarshjalaja** [reply]
thanks a lot. will try this.
